src/index.html

```diff
@@ -17,17 +18,32 @@
     -->
     <script src="/zone.min.js"></script>
 
-    <!--
-      For development mode, this script will be bundled on-the-fly by
-      the devserver, based on the latest JS files built by Bazel.
-      For production, you should be able to use the same index.html file, but
-      this JS resource would be served from a static file.
+    <script src="/system.js"></script>
+
+    <script>
```

Comment:
looks like this script could easily be another output of the rollup_bundle rule, placed in the runfiles.transitive_files so it's available to be loaded by anything with a transitive data[] dep